include/c2pa.hpp

```diff
@@ -43,21 +60,29 @@ namespace C2pa
     };
 
     // Return the version of the C2pa library
-    string version()
+    static string version()
     {
         auto result = c2pa_version();
         string str = string(result);
         c2pa_release_string(result);
         return str;
     }
 
+    /// Load c2pa settings from a string in a given format
```

Comment:
I think C2PA should be capitalized in all doc comments unless depicting a variable name.